### PR TITLE
cmake: Fix MSVC runtime selection for win-dshow and win-capture targets

### PR DESCRIPTION
**Pull request by PatTheMav**
### Description
Adds the missing explicit MSVC runtime selection to the the 32-bit `win-capture` target and also moves the same selection out of the shared virtual cam interface library to the actual architecture-specific targets.

### Motivation and Context
The `MSVC_RUNTIME_LIBRARY` target property when set on an `INTERFACE` type library is not inherited by targets which consume the library, thus their property is set automatically set by CMake.

This affected the virtual camera target which uses an `INTERFACE` library for sources and settings shared between its 64-bit and 32-bit variants.

The PR sets the property directly on the "final" output targets to fix this.

### How Has This Been Tested?
Needs to be tested on an Intel-based Windows installation.

### Types of changes
- Bug fix (non-breaking change which fixes an issue)

### Checklist:
- [x] My code has been run through [clang-format](https://github.com/obsproject/obs-studio/blob/master/.clang-format).
- [x] I have read the [**contributing** document](https://github.com/obsproject/obs-studio/blob/master/CONTRIBUTING.rst).
- [x] My code is not on the master branch.
- [x] The code has been tested.
- [x] All commit messages are properly formatted and commits squashed where appropriate.
- [x] I have included updates to all appropriate documentation.

### CODE REVIEW

**Comment by PatTheMav** [comment]
@derrod could you check the artifact once it's done?

**Review by RytoEX — CHANGES_REQUESTED** [review]
This also needs to update `plugins/win-capture/get-graphics-offsets/cmake/32bit-build.cmake` and `plugins/win-capture/inject-helper/cmake/32bit-build.cmake`.

**Comment by PatTheMav** [comment]
> This also needs to update `plugins/win-capture/get-graphics-offsets/cmake/32bit-build.cmake` and `plugins/win-capture/inject-helper/cmake/32bit-build.cmake`.

Done.

File: plugins/win-capture/inject-helper/cmake/32bit-build.cmake [hunk 1]
```diff
@@ -2,4 +2,7 @@ project(inject-helper)
 
 add_executable(inject-helper)
 target_link_libraries(inject-helper PRIVATE _inject-helper)
-set_property(TARGET inject-helper PROPERTY OUTPUT_NAME inject-helper32)
+
+# cmake-format: off
+set_target_properties_obs(inject-helper PROPERTIES OUTPUT_NAME inject-helper32 MSVC_RUNTIME_LIBRARY "MultiThreaded$<$<CONFIG:Debug>:Debug>")
```
Comment (RytoEX) [inline plugins/win-capture/inject-helper/cmake/32bit-build.cmake:7]: Maybe `set_target_properties`?

**Review by RytoEX — CHANGES_REQUESTED** [review]

File: plugins/win-capture/inject-helper/cmake/32bit-build.cmake (hunk 1 above)
Comment (PatTheMav) [reply plugins/win-capture/inject-helper/cmake/32bit-build.cmake:7]: Ugh, correct.

**Comment by RytoEX** [comment]
Looks correct at a glance. Will double check later.

**Review by RytoEX — APPROVED** [review]
Manually checked the properties of each affected VS project, both 32-bit and 64-bit, and this seems correct now.

**Review by derrod — APPROVED** [review]
Confirmed using PEStudio that binaries now no longer try to load VC runtime libraries.

**Review by WizardCM — APPROVED** [review]
Confirmed, this fixes both issues on a system that doesn't have the 32bit redistributables installed.

**Pull request merged**